www/js/menu.js

```diff
@@ -329,6 +330,23 @@ module.exports = {
 
   showNotificationsModal,
 
+  showDisplayTab(fromMainMenu) {
+    search.activateNavLink('display', true);
+    search.activateNavPage('session', { id: 'display', label: 'Display' });
+
+    const isPresenterView = document.body.classList.contains('presenter-view');
+    let settingsClickSource = isPresenterView ? 'from_presenter_view' : 'not_from_presenter_view';
+    if (fromMainMenu) settingsClickSource += '_main_menu';
+    analytics.trackEvent('newSettings', settingsClickSource);
+
+    const sessionPage = document.querySelector('#session-page');
+    sessionPage.classList.add('bounce-animate');
+    sessionPage.offsetWidth = undefined; // reflow to restart animation
```

Comment:
what does this do? seems like a hack.

Reply:
It is used to restart CSS animation on click. You are right, it's a bit of a hack. I couldn't find a nice another way to restart CSS animations and I didn't want to use setTimeout just for the sake of this.